stdlib/ballerina-builtin/src/main/ballerina/auth/Package.md

```diff
@@ -4,21 +4,21 @@ This package provides a set of default authentication providers that can be exte
 
 ### Authentication Provider
 
-An authentication provider defines an authentication scheme that could be used to protect endpoints. The `auth:AuthProvider` type acts as the interface for all the authentication providers. Any type of implementation, such as LDAP, JDBC, and file based, should be object-wise similar. 
+An authentication provider defines an authentication scheme that could be used to protect endpoints. The `auth:AuthProvider` type acts as the interface for all the authentication providers. Any type of implementation, such as LDAP, JDBC, and file based, should be similar object-wise. 
 
-By default there are two implementations of that. They are, `auth:ConfigAuthProvider` which authenticates based on usernames and passwords stored in ‘ballerina.conf’, and `auth:JWTAuthProvider` which authenticates by validating a JWT. It is possible to implement more such authentication mechanisms.
+By default, there are two implementations of the `auth:AuthProvider`. They are, the `auth:ConfigAuthProvider`, which authenticates based on usernames and passwords stored in the `ballerina.conf` file, and the `auth:JWTAuthProvider`, which authenticates by validating a JWT. It is possible to implement more such authentication mechanisms.
```

Comment:
Shall we not name the config file? `ballerina.conf` is just the name of the default config file. In reality, the configuration file can be named anything the user desires. How about simply referrung to it as a configuration file?

Reply:
changed this line with commit https://github.com/ballerina-platform/ballerina-lang/pull/8263/commits/9de8db89d6e6b8b694d1ab18b4315271a8639995

Now it reads as:
"They are, the `auth:ConfigAuthProvider`, which authenticates based on usernames and passwords stored in a configuration file, and the `auth:JWTAuthProvider`, which authenticates by validating a JWT. It is possible to implement more such authentication mechanisms. The default configuration file is named `ballerina.conf`. " 

Added the last line because it might be helpful to know what the default config file is.

Reply:
Oh okay. In that case, it's not really required to mention it here I guess. Removed it with commit https://github.com/ballerina-platform/ballerina-lang/pull/8263/commits/15a67321401f8e2ef3cea6b81296feff09796a0e.

@dulanjal please review.